src/mysql_k8s_helpers.py

```diff
@@ -253,6 +261,85 @@ def create_custom_config_file(self, report_host: str) -> None:
         except Exception:
             raise MySQLCreateCustomConfigFileError()
 
+    def execute_backup_commands(
+        self,
+        s3_bucket: str,
+        s3_directory: str,
+        s3_access_key: str,
+        s3_secret_key: str,
```

Comment:
I think we should pass `endpoint` as the bare minimum in this PR.
later, in a separate PR/ticket, we can add `region`, `s3-api-version`, `s3-uri-style`, `storage-class`.

Reply:
addressesd in aedeff6b9921c57431722fcf5a827a4509226ae3

Reply:
can you please create a ticket for the rest parameters and a comment link in response to this message?

Reply:
oops, yes. created ticket [DPE-1229](https://warthogs.atlassian.net/browse/DPE-1229)